cmd/tapcli/assets.go

```diff
@@ -291,11 +315,10 @@ func mintAsset(ctx *cli.Context) error {
 	return nil
 }
 
-var finalizeBatchCommand = cli.Command{
-	Name:        "finalize",
-	ShortName:   "f",
-	Usage:       "finalize a batch",
-	Description: "Attempt to finalize a pending batch.",
+var fundBatchCommand = cli.Command{
+	Name:        "fund",
+	Usage:       "fund a batch",
+	Description: "Attempt to fund a pending batch.",
```

Comment:
Add some additional context here similar to my previous comment about _when_ this might be needed? Otherwise the additional commands might confuse the users.